manifests/bifrost/bifrost.yaml

```diff
@@ -65,8 +65,13 @@ spec:
     name: bifrost
   ports:
   - port: 80
+    name: bifrost
     protocol: TCP
     targetPort: 8087
+  - port: 8099
+    name: service
```

Comment:
@gabrieljackson can we rename service here and in the service monitor to be called metrics?

Reply:
I called it service because that is what it is called internally in bifrost as it is used for metrics and health endpoints (and maybe more in the future).

Still want me to change it?

Reply:
Ah if its not dedicated to only metrics lets leave it like that ;)